gnucash/report/standard-reports/net-charts.scm

```diff
@@ -263,6 +263,72 @@
     (define (monetary->double monetary)
       (gnc-numeric-to-double (gnc:gnc-monetary-amount monetary)))
 
+    (define (split->date s)
+      (xaccTransGetDate (xaccSplitGetParent s)))
+
+    ;; this function will scan through the account splitlist, building
+    ;; a list of balances along the way. it will use the dates
+    ;; specified in the variable dates-list.
+    ;; input: account
```

Comment:
Any reason not to specify dates-list as a parameter to the function rather than assuming it's available in the parent scope ?

Reply:
> Any reason not to specify dates-list as a parameter to the function rather than assuming it's available in the parent scope ?
> Oh never mind. The function is meant to be called in a map context, which only takes one argument...

Yeah trying to be elegant for map... :)

Later on when exporting this function I'll likely be creating a wrapper for it... I'd like to call this xaccAccountGetBalancesAsOfDates but this wouldn't be consistent with other scm functions :-/